test/test_utils.py

```diff
@@ -954,6 +954,88 @@ def test_escape_url(self):
         )
         self.assertEqual(escape_url('http://vimeo.com/56015672#at=0'), 'http://vimeo.com/56015672#at=0')
 
+    def test_js_to_json_vars_strings(self):
+        def parse_js_to_json(js, vars):
+            try:
+                return json.loads(js_to_json(js, vars))
+            except json.JSONDecodeError as e:
+                raise ValueError(f'JSON: {e.msg}: line {e.lineno} column {e.colno} "{e.doc}"')
+
+        self.assertDictEqual(
+            parse_js_to_json(
+                '''{
+                    'null': a,
+                    'nullStr': b,
+                    'true': c,
+                    'trueStr': d,
+                    'false': e,
+                    'falseStr': f,
+                }''',
+                {
+                    'a': 'null',
+                    'b': '"null"',
+                    'c': 'true',
+                    'd': '"true"',
+                    'e': 'false',
+                    'f': '"false"',
+                }
```

Comment:
The tests are wrong? This should be
```suggestion
                {
                    'a': null,
                    'b': 'null',
                    'c': True,
                    'd': 'true',
                    'e': False,
                    'f': 'false',
                }
```

Reply:
These are the variables that are passed to `js_to_json` as a dict. The values are what would have been extracted from a JS string, so `a` and `b` could come from `someFunction(null, "null")`.